### ISSUE TITLE
SMTP sender

### ISSUE THREAD

**Issue by m-paniez9292** [octobercms/october#2228]
##### Expected behavior
October sends email

##### Actual behavior
I have a time out connection : "Connection could not be established with host smtp.domain.com [Connection timed out #110]" on line 269 of ...../vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/StreamBuffer.php
I have another site with the same smtp sender, but with october v316, and it works nice.
I try SSL and TLS support, but same issue

##### Reproduce steps
In settings > Email > configuration, choose smtp sender

##### October build
Latest : v349

**Comment by shina** [comment]
Are you using the same server?
It usually happens on Digital Ocean servers

**Comment by m-paniez9292** [comment]
Re,
These websites are on differents servers and different php version : 
- october v316 (which works fine) are on debian 7 and php 5.4.45
- october v349 (which have problem) are on debian 8 and php 7.

Maybe it's the php version ..??

Thanks

**Comment by shina** [comment]
Well... I'm not familiar with php7 yet, but I believe that it's not the problem.

It's looks like a configuration issue.
Check with your server provider if they are blocking the smtp ports to prevent spam attacks.

ps: I'm assuming that your smtp configuration is correct

**Comment by m-paniez9292** [comment]
Re,
After more tests, with tcpdump on smtp port, I can see my emails tests go out from our server to mail server (like IP <http_serverIP> > <mail_serverIP> ....
I can do more tests if you want (I try with our mail server AND a gmail account).
Thanks

**Comment by daftspunk** [comment]
This is most definitely a configuration issue.

```
Connection could not be established with host smtp.domain.com 
```

The error shows October functions as normal. The web server is unable to open a socket to the SMTP server, it is timing out, perhaps because the SMTP port is blocked by the firewall?

**Comment by m-paniez9292** [comment]
Re,
After more and more tests, I found my way and October works great.
Sorry about that.
Thanks

**Comment by daftspunk** [comment]
No problem! Feel free to share your solution in case others encounter this issue as well.

**Comment by MichiReich** [comment]
How you figured it out?

**Comment by daftspunk** [comment]
I think the issue was Gmail rewriting the sender field internally as part of their SMTP server. Please report a new issue if this occurs for anyone else and we can troubleshoot it.
